### PR TITLE
Enable allure dev reporting

### PR DESCRIPTION
**Pull request by PatStLouis**
This enables local allure report generation for developers.

### CODE REVIEW

**Review by aljones15 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -326,6 +327,19 @@ you can check the
 [vc-test-suite-implementations](https://github.com/w3c/vc-test-suite-implementations)
 library.
 
+### Allure Reporting
+It's also possible to generate local allure reports for analyzing and debuging results.
```
Comment (davidlehn) [inline README.md:331]: Can you add a link to "allure".  I suspect there will be people (like me) that don't know what that is.

File: README.md [hunk 2]
```diff
@@ -326,6 +327,19 @@ you can check the
...
+
+To run the tests and browse the report, use the following commands:
+```bash
+# Running the tests
+npx mocha tests/
+
+# Running the reporting server
+allure serve allure-results
+
```
Comment (davidlehn) [inline README.md:340]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -326,6 +327,19 @@ you can check the
...
+### Allure Reporting
+It's also possible to generate local allure reports for analyzing and debuging results.
+
+To run the tests and browse the report, use the following commands:
+```bash
+# Running the tests
+npx mocha tests/
+
+# Running the reporting server
+allure serve allure-results
```
Comment (davidlehn) [inline README.md:339]: Should this be run with `npx`?

File: README.md (hunk 1 above)
Comment (TallTed) [inline README.md:331]:
```suggestion
It's also possible to generate local allure reports for analyzing and debugging results.
```

**Review by BigBlueHat — APPROVED** [review]
+1 modulo the recommendations from others.

File: README.md (hunk 3 above)
Comment (PatStLouis) [reply README.md:339]: It's not necessary, the `allure-commandline` package provide a direct commandline access to the allure package:
https://www.npmjs.com/package/allure-commandline

There's 2 npm packages installed here, `allure-mocha` and `allure-commandline`. `allure-mocha` is the mocha specific reporter that will output the results into a directory. The `allure-commandline` is the allure software that will build the report and host a local reporting server.

File: README.md (hunk 1 above)
Comment (PatStLouis) [reply README.md]: added a link a short description for the allure reporting framework

File: README.md (hunk 2 above)
Comment (PatStLouis) [reply README.md]: Removed the newline within the bash codeblock

**Pull request merged**